justfile

```diff
@@ -23,3 +23,46 @@ test-init-image:
 # Exec init image
 exec-init-image:
 	docker run --rm --name pid --tty --interactive pid1runner sh
+
+# Build binary for other architectures
+binaries clean='false':
+	cross build --target x86_64-unknown-linux-gnu --release
```

Comment:
This just recipe has various architecture targets for which binary will be crated. The major ones which we will be using is x86_64 and ARM 64. 

I have added some other musl based architectures based on what the project `tini` supports. Do let me know if you want me to add anything else or remove something from here. This was also first time I'm tinkering with the `cross` tool to build static binary.

Reply:
Looks like an extensive list!